src/main/java/Bulldozer.java

```diff
@@ -0,0 +1,9 @@
+class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Truck started working.");
```

Comment:
The message should indicate that it is a `Bulldozer` that started working, not a `Truck`. Ensure your output message matches the class it's in to avoid confusion.